process/templates/SIG_NAME_graduation_stage.md

```diff
@@ -1,5 +1,32 @@
 ## Special Interest Group (SIG) graduation
 
 The governing body must agree that the SIG has completed a major deliverable
-
   * Link to relevant documentation.
+
+## SIG has met all Incubating requirements
+  * "link to Incubating PR"
+
+### Governance
+SIG must have documented project governance and be able to demonstrate that governance in action.
+  * "link to governance documents"
+SIG has met at least 4 times over a period of at least 2 months since becoming incubating
+  * "link to meeting agenda"
+Have a define roadmap and annual goals for the project
+  * "link to roadmap and goals"
+SIG has completed a major deliverable
+	  * "link to completed deliverable"
+
+### To retain Graduated status
+A TI is expected to continue operating per the above guidelines, and to provide the TAC with quarterly status updates, and to approach the TAC when seeking approval of substantial changes (such as when accepting or promoting new Projects).
+
+### SIG References
+The SIG should provide a list of existing resources with links to the repository, and if available, website, a roadmap, demos and walkthroughs, and any other material to showcase the existing breadth, maturity, and direction of the SIG.
+ Reference              | URL |
+|-----------------------|-----|
+| Repo                  |     |
+| Meeting Agenda        |     |
+| OSSF Calendar Entry   |     |
+| Website               |     |
+| Security.md           |     |
+| Roadmap               |     |
```

Comment:
These may not apply to SIGs.

Reply:
we want all LF repos to have a security.md.  we can never predict how someone might engage us, and the generic file (that I think already has been pushed out everywhere) is just a generic statement that points to the LF to report to.  SIGs may want to have a roadmap to document their future activities (for long-lived SIGs like EDU, for example)

Reply:
Makes sense, let's document this! I can add this explanation to the SIG lifecycle doc while I'm making edits there as part of #296 .